src/main/java/ooss/Student.java

```diff
@@ -1,4 +1,26 @@
 package ooss;
 
-public class Student {
+public class Student extends Person {
+    private Klass klass;
+    public Student(int id, String name, int age) {
+        super(id, name, age);
+    }
+    @Override
+    public String introduce(){
+        if (this.klass.isLeader(this))
```

Comment:
can check klass is null or not first, otherwise will return nullPointerException